Open_Weather-in-hook-and-redux/src/types/Weather.ts

```diff
@@ -0,0 +1,5 @@
+export interface Weather {
+    name?: string;
+    main: { temp: number, feels_like: number, pressure: number, humidity: number };
+    weather: [{ icon: any }];
```

Comment:
тут скорее всего ты хотел сделать массив объектов, это будет так:
```suggestion
    weather: { icon: string }[];
```